sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeITCluster/teardown.sh

```diff
@@ -0,0 +1,24 @@
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+#!/bin/bash
+
+set -e
+
+# Delete Cassandra services and statefulset.
+kubectl delete -f cassandra-service.yaml
+kubectl delete -f cassandra-external-service.yaml
+kubectl delete -f cassandra-statefulset.yaml
```

Comment:
when I'm running teardown and then setup again, I see the error:
Failed to attach volume "..." on node "..." with: googleapi: Error 400: The disk resource 'projects/.../zones/us-central1-a/disks/...' is already being used by 'projects/.../zones/us-central1-a/instances/...

I earlier asked you to remove the disks and switch to normal disks, so that might resolve this issue, but if not,

either way, make sure you can do a full setup -> teardown -> setup cycle